lib/tasks/create_missing_whitehall_assets.rake

```diff
@@ -0,0 +1,53 @@
+# rubocop:disable Metrics/BlockLength
+task create_missing_whitehall_assets: :environment do
+  placeholder_path = Rails.root.join('tmp/whitehall-attachment-placeholder.txt')
```

Comment:
Minor: Using the word "placeholder" might be confusing, because it's used in both Whitehall and Asset Manager for the target of the redirects that happen if an asset has not yet been virus scanned. Perhaps something like `replaced-whitehall-attachment` would be better...?

Reply:
This Rake task is going away as soon as it's been run so I'm not worried about the potential confusion.

Reply:
While the Rake task is going away the file content will stay on S3, no?

Reply:
I think the "whitehall-attachment-placeholder.txt" filename will only be present in the database, and there were only 19 assets created with this filename so I'm still not too worried about the potential confusion.